packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/iOS/signin.md

```diff
@@ -14,37 +14,30 @@ let config = OktaOidcConfig(/* plist */)
 let oktaOidc = OktaOidc(configuration: config)
 ```
 
-Then, you can start the authorization flow by simply calling `signInWithBrowser`. When successfully authorized, this operation returns a valid `OktaOidcStateManager` instance in its callback:
+Start the authorization flow by simply calling `signInWithBrowser` on `OktaOidc`. After a successful sign-in, this operation returns a valid `OktaOidcStateManager` instance in its callback:
 
 ```swift
 oktaOidc.signInWithBrowser(from: self) { stateManager, error in
     if let error = error {
         // Error
         return
     }
-
-    // stateManager.accessToken
-    // stateManager.idToken
-    // stateManager.refreshToken
+    
+    // #1 Store instance of stateManager into the iOS keychain
+    stateManager.writeToSecureStorage()
+
+    // #2 Use tokens
+    print(stateManager.accessToken)
+    print(stateManager.idToken)
+    print(stateManager.refreshToken)
 }
 ```
-You are responsible for storing the state manager. You can securely store the `OktaOidcStateManager` instance in the iOS keychain:
+
+To restore `OktaOidcStateManager` object from iOS keychain do the following:
```

Comment:
To restore the `OktaOidcStateManager` object from the iOS keychain, do the following: